README.md

```diff
@@ -40,7 +40,7 @@ Give a ⭐️ if you like this project!
 
```

Comment:
- [ **OPTIONAL** ] **RECOMENDED:** **In the README.md file:** 
I think it would be better if you add all the project's **Authors contributors** in the  **## Authors** section with their specific information to make your documentation looks more professional i.e.:
```
## Authors

👤 **Tarik Bouari**
- GitHub: [@tarikbouari](https://github.com/tarikbouari))
- Twitter: [@tarikbouari](https://twitter.com/TarikBouari)
- LinkedIn: [tarikbouari](https://www.linkedin.com/in/tarik-bouari-44b7191a6

👤 **Author_2_name**
- GitHub: [@username](https://github.com/username))
- Twitter: [@username](https://twitter.com/username)
- LinkedIn: [username](https://www.linkedin.com/in/username

👤 **Author_3_name**
- GitHub: [@username](https://github.com/username))
- Twitter: [@username](https://twitter.com/username)
- LinkedIn: [username](https://www.linkedin.com/in/username

...
```